faq.html

```diff
@@ -680,16 +682,14 @@ <h3>If I reinstall Blixt Wallet or switch devices, can I keep my previous channe
 
   <section id="blixt-lnaddress">
     <h3>Does Blixt Wallet have a Lightning Address I can receive with?</h3>
-    <p>NO, NOT right now. But soon it will have added a new powerful feature named <a href="https://github.com/hsjoberg/lightning-box">Lightning Box"</a></p>
-    <p>This feature will provide the user with a personalized self custodial Lightning Address linked to his own Blixt wallet.</p>
-    <p>Stay tuned, soon will be released this new feature for Blixt!</p>
-    <p>Can I receive sats while Blixt wallet is closed?</p>
-    <p>Right now, no. Soon, when the Lightning Box will be implemented in Blixt, then yes, you will be able to receive, while your Blixt app is closed / not online</p>
+    <p>Yes, starting with <a href="https://github.com/hsjoberg/blixt-wallet/releases/tag/v0.6.9-420">Blixt v0.6.9-420</a> you can have 
+      a Lightning Address of your own.</p>
+    <p>See more details in the Blixt Github release notes and/or in the <a href="/features#blixt-drawer">"Blixt Features" page</a>.</p>
   </section>
 
   <section id="blixt-dunder-channels">
     <h3>Channels opened with Dunder are max 400k sats capacity. How can I have larger Dunder channels opened?</h3>
-    <p>Yes, the current limit for opening Dunder channels with Blixt node is max 400k sats.</p>>
+    <p>Yes, the current limit for opening Dunder channels with Blixt node is max 400k sats.</p>
```

Comment:
I decreased it to 200k

```suggestion
    <p>Yes, the current limit for opening Dunder channels with Blixt node is max 200k sats.</p>
```